app/components/UI/LedgerModals/LedgerConfirmationModal.test.tsx

```diff
@@ -0,0 +1,13 @@
+import { shallow } from 'enzyme';
+import { renderScreen } from '../../../util/test/renderWithProvider';
+import LedgerConfirmationModal from './LedgerConfirmationModal';
+
+
+describe('LedgerConfirmationModal', () => {
+  it('should render correctly', () => {
```

Comment:
I suggest gloably renaming these kind of snapshot tests as it doesn't check if it renders correctly but only if nothing has changed since last snapshot. it can be totally wrong and pass if not changed.
```suggestion
  it('render matches latest snapshot', () => {
```